client/src/modules/dashboard/Events/pages/EventsPage.tsx

```diff
@@ -248,7 +249,11 @@ export const EventsPage: NextPageWithLayout = () => {
                         {name}
                       </LinkButton>
                     </VStack>
-                    <Text>{invite_only ? 'Yes' : 'No'}</Text>
+                    {invite_only ? (
+                      <LockIcon> invite_only</LockIcon>
+                    ) : (
+                      <UnlockIcon>Open for attending</UnlockIcon>
+                    )}
```

Comment:
It might be not needed to change that here. Difference between icons, and at that size is not easily noticeable. There's also more than enough space for the text written _Yes_/_No_. Maybe leave icon only for the invite only events? To keep it consistent with other pages.